docs/pages/ds-video-port_TestSpec.md

```diff
@@ -0,0 +1,182 @@
+
+Unplug the Video port,Verify the surround mode
+
+### Check Video Content Format and HDR Capability
+
+|Test Functionality|Description|L2|L3|Source|Sink|
+|------------------|-----------|--|--|------|----|
+|Check Video Format Content and HDR Capability|Notify an event when the list of video Format changes|`NA`|`Y`|`Y`|`Y`|
+||Get the HDR capabilities and status|Y|`NA`|`Y`|`Y`|
+||Set Force Disable 4KSupport with/without playback|`Y`|`Y`|`Y`|`Y`|
+||Disable 4KSupport with external analyzer|`NA`|`Y`|`Y`|`NA`|
+||Reset the video output to SDR with/without playback|`NA`|`Y`|`Y`|`Y`|
+||Reset the video output to SDR with external analyzer|`NA`|`Y`|`Y`|`NA`|
+
+#### Test Startup Requirement-Check Video Content Format and HDR Capability
+
+Playback the pre-define streams
+
+#### Emulator Requirements-Check Video Content Format and HDR Capability
+
+[Emulator Requirements](#emulator-requirements)
+
+#### Control Plane Requirements-Check Video Content Format and HDR Capability
+
+Check the port output resolutions and HDR with analyzer
+
+### Check Video Resolution
+
+|Test Functionality|Description|L2|L3|Source|Sink|
+|------------------|-----------|--|--|------|----|
+|Check Resolution with/without playback|Set/Get Video port properties like pixel resolution, Aspect ratio, Stereo Scopic modes, frame rates and scan modes|`Y`|`Y`|`Y`|`NA`|
+||Get Video port properties like pixel resolution, Aspect ratio, Stereo Scopic modes, frame rates and scan modes and verify|`Y`|`NA`|`NA`|`Y`|
+||Check Video port properties with external Analyzer|`NA`|`Y`|`Y`|`NA`|
+
+#### Test Startup Requirement-Check Video Resolution
+
+Playback the pre-define streams
+
+#### Emulator Requirements-Check Video Resolution
+
+[Emulator Requirements](#emulator-requirements)
+
+#### Control Plane Requirements-Check Video Resolution
+
+Verify the AspectRatio,video Stereo Scopic modes,video Frame rates,interlaced/progressive.
+
+### HDCP and DTCP Management
+
+|Test Functionality|Description|L2|L3|Source|Sink|
+|------------------|-----------|--|--|------|----|
+|Check HDCP and DTCP Management|Check enable/disable the DTCP/HDCP for the specified video port|`Y`|`Y`|`Y`|`NA`|
+||Check DTCP/HDCP status for valid port|`Y`|`Y`|`Y`|`Y`|
+||Check HDCP protocol status|`Y`|`Y`|`Y`|`Y`|
+||Ignore EDID status|`N`|`Y`|`Y`|`NA`|
+||Notify event if the HDCP status change|`NA`|`Y`|`Y`|`NA`|
+||Check HDCP/DTCP status with external analyzer|`NA`|`Y`|`Y`|`NA`|
+
+#### Test Startup Requirement-HDCP and DTCP Management
+
+Playback the pre-define streams
+
+#### Emulator Requirements-HDCP and DTCP Management
+
+[Emulator Requirements](#emulator-requirements)
+
+#### Control Plane Requirements-HDCP and DTCP Management
+
+Check the HDCP/DTCP status with external analyzer
+
+### Color Capabilities
+
+|Test Functionality|Description|L2|L3|Source|Sink|
+|------------------|-----------|--|--|------|----|
+|Check Color Capabilities|Set/Get Color Space and compare with the configuration file|`Y`|`Y`|`Y`|`Y`|
```

Comment:
There is no set for colordepth / color space

Reply:
Updated